docs/Rx.md

```diff
@@ -167,6 +167,8 @@ Use a diode with cathode to receiver serial rx output (for example 1N4148),
 the anode is connected to the FC serial _TX_ pin, and also via a 
 resistor (10KOhm) to the receiver ibus sensor port.
 
+Note (2018-07-27): In some cases, the value of the series resistor may be too large, and going down to 1K[ohm] may provide a good result.
```

Comment:
This seems to be missing a 'N'.

Reply:
Lol, my bad. Turns out the latest version of the 'Refined GitHub' Chrome extension has a bug cutting off the first character of every changed line.